docs/UserGuide.md

```diff
@@ -65,85 +63,171 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
 
 ### Viewing help : `help`
 
-Shows a message explaning how to access the help page.
+Shows a message explaining how to access the help page.
 
 ![help message](images/helpMessage.png)
 
 Format: `help`
 
 
-### Adding a person: `add`
+### Adding a patient: `add_p`
 
-Adds a person to the address book.
+Adds a patient to the system.
 
-Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]…​`
+Format: `add_p n/NAME ic/NRIC p/PHONE_NUMBER`
 
-<div markdown="span" class="alert alert-primary">:bulb: **Tip:**
-A person can have any number of tags (including 0)
-</div>
+Examples:
+* `add_p n/John Doe ic/T0212385J p/98765432`
+* `add_p ic/S9912343G n/Betsy Crowe p/81235833`
+
+### Listing all patient : `list_p`
+
+Shows a list of all patients in the system.
+
+Format: `list_p`
+
+### Editing a patient : `edit_p`
+
+Edits an existing patient in the system.
+
+Format: `edit_p ic/NRIC [n/NAME] [p/PHONE]`
+
+* Edits the person with the specified `NRIC`.
+* At least one of the optional fields must be provided.
+* The NRIC of the patient cannot be changed.
```

Comment:
the original AB3 does it by asking for the INDEX of the person on the list instead of a particular "attribute" of an individual to select the person to edit. this discussion will be deferred to v1.2